lib/internal/navigator.js

```diff
@@ -18,10 +18,12 @@ const {
 } = internalBinding('os');
 
 const kInitialize = Symbol('kInitialize');
+const nodeVersion = process.version;
 
 class Navigator {
   // Private properties are used to avoid brand validations.
   #availableParallelism;
+  #userAgent = `Node.js/${nodeVersion.slice(1, nodeVersion.indexOf('.'))}`;
```

Comment:
I would prefer we didn’t, because it’s not really correct to publish 0.0 when we’re not those minor/patch versions. I think Chrome did so in order to avoid breaking people who are parsing `userAgent` with regex, but we don’t have the backward compatibility concern.

Reply:
> #userAgent = `Node.js/${process.versions.node.slice(0,2)}.0.0`;

This would break at Node v100.